clinica/utils/dwi.py

```diff
@@ -330,6 +330,7 @@ def hmc_split(in_file, in_bval, ref_num=0, lowbval=5.0):
 
 
 def check_dwi_volume(in_dwi, in_bvec, in_bval):
+    # TODO return str instead of raising an error, so that user can be informed of which file is causing problem
```

Comment:
In upcoming new version `dwi-preprocessing`, it will be done at `init_node` pipeline. With this solution, compliant DWI files will run even though this is not the case for "non-compliant" DWI files.

Reply:
Ok